README.md

```diff
@@ -134,74 +135,115 @@ In the example above, the sidecar will run only the Apache plugin. Additionally,
 
 ## Configuration
 
+### Enabling Docker Logs collector from Kubernetes nodes
+
+In this configuration SolarWinds Snap Agent DaemonSet will gather Docker logs from underlaying node and publish them to Loggly (in addition to gathering HostAgent and Docker metrics to AppOptics).
+This option is not enabled by default, it have to be turn on to start working. 
+
+
+* Create `solarwinds-token` secret in your cluster. To create it run:
+
+  ``` bash
+  kubectl create secret generic solarwinds-token -n kube-system --from-literal=SOLARWINDS_TOKEN=<REPLACE WITH TOKEN>
+  ```
+
+* (Optional step) If your token for Loggly is different than your SolarWinds token, please create new Kubernetes secret. If the tokens are the same, there is no need to perform this step - in that case `SOLARWINDS_TOKEN`, will be used by Loggly Publisher plugin.
+
+  ``` bash
+  kubectl create secret generic loggly-token -n kube-system --from-literal=LOGGLY_TOKEN=<REPLACE WITH LOGGLY TOKEN>
```

Comment:
add PAPERTRAIL_TOKEN note also

Reply:
Added general info regarding  PAPERTRAIL_TOKEN, and info regarding  PAPERTRAIL_TOKEN in relevant places.